### PR TITLE
E1685: UI changes for review and score reports

### PR DESCRIPTION
**Pull request by coder-neo**
Tasks completed:
1. Updated the UI for review report in instructor's view.
2. Added new view in response controller to be displayed when clicked on any review in the alternate view.
3. Create a tabbed view for each review view type : Heat map, Overall statistics, Individual reviews
4. Link to show/hide all review for a particular round.
5. Link to show/hide question,score, comments for individual review.

### CODE REVIEW

File: app/assets/javascripts/tableactions.js [hunk 1]
```diff
@@ -1,79 +1,98 @@
...
-      }
-    }
-    for (i=0; i < displayedRows.length; i++){
-      if (i % 2 == 0){
-        displayedRows[i].className = "even";        
-      } else {
-        displayedRows[i].className = "odd";
-      }           
+function alternate(id) {
+    if (document.getElementsByTagName) {
```
Comment (yangsong8) [inline app/assets/javascripts/tableactions.js]: White space change only?

File: app/controllers/response_controller.rb [hunk 2]
```diff
@@ -148,6 +148,12 @@ def view
     set_content
   end
 
+  def view_review
```
Comment (yangsong8) [inline app/controllers/response_controller.rb]: Is this controller method newly added? What's the reason, plz?

**Comment by yangsong8** [comment]
This is a comment for grading purpose ONLY, do not change your code unless you r told to.

On the "Alternate view" tab, the option of giving review score is available. This means that a student can give him/herself review score on "view_my_scores" page.

File: app/controllers/response_controller.rb (hunk 2 above)
Comment (coder-neo) [inline app/controllers/response_controller.rb]: Added new view in response controller to be displayed when clicked on any review in the alternate view.

**Pull request closed without merging**